packages/relay/src/lib/constants.ts

```diff
@@ -140,11 +140,11 @@ export default {
   },
 
   // TODO: Replace with actual values - https://github.com/hashgraph/hedera-json-rpc-relay/issues/2895
-  HBAR_RATE_LIMIT_DURATION: parseInt(process.env.HBAR_RATE_LIMIT_DURATION || '80000'), // 80 seconds
-  HBAR_RATE_LIMIT_TOTAL: BigNumber(process.env.HBAR_RATE_LIMIT_TINYBAR || '11000000000'), // 110 HBARs per 80 seconds
-  HBAR_RATE_LIMIT_BASIC: BigNumber(process.env.HBAR_RATE_LIMIT_BASIC || '92592592'), // Equivalent of 1000 HBARs per day
-  HBAR_RATE_LIMIT_EXTENDED: BigNumber(process.env.HBAR_RATE_LIMIT_EXTENDED || '925925925'), // Equivalent of 10000 HBARs per day
-  HBAR_RATE_LIMIT_PRIVILEGED: BigNumber(process.env.HBAR_RATE_LIMIT_PRIVILEGED || '1851851850'), // Equivalent of 20000 HBARs per day
+  HBAR_RATE_LIMIT_DURATION: parseInt(process.env.HBAR_RATE_LIMIT_DURATION || '86400000'), // 1 day
+  HBAR_RATE_LIMIT_TOTAL: BigNumber(process.env.HBAR_RATE_LIMIT_TINYBAR || '800000000000'), // 8000 HBARs
```

Comment:
Here we use `HBAR_RATE_LIMIT_TOTAL` but in the doc it's `HBAR_RATE_LIMIT_TINYBAR`. I like `HBAR_RATE_LIMIT_TOTAL` better.

Reply:
Unfortunately we can't really rename or replace configs easily due to backwards compatibility.
We should actually change this back `HBAR_RATE_LIMIT_TINYBAR` if `HBAR_RATE_LIMIT_TOTAL` was never used before.
The issue is if an operator has configured it today and upgrades we don't wan to break them by no longer respecting the previous config.

Reply:
Right yeah that makes sense!